packages/graphql/tests/integration/directives/fulltext.int.test.ts

```diff
@@ -74,672 +94,2415 @@ describe("@fulltext directive", () => {
+                        name: person1.name,
+                    },
+                },
+                {
+                    [personTypeLowerFirst]: {
+                        name: person2.name,
+                    },
+                },
+            ]);
+        });
+        test("Nodes aren't returned if not requested", async () => {
+            const query = `
+                query {
+                    ${queryType}(phrase: "some name") {
+                        score
+                    }
+                }
+            `;
+            const gqlResult = await graphql({
+                schema: generatedSchema,
+                source: query,
+                contextValue: {
+                    driver,
+                    driverConfig: { database: databaseName },
+                },
+            });
+
+            expect(gqlResult.errors).toBeFalsy();
+            expect((gqlResult.data?.[queryType] as any[])[0][personTypeLowerFirst]).toBeUndefined();
+            expect((gqlResult.data?.[queryType] as any[])[1][personTypeLowerFirst]).toBeUndefined();
+            expect((gqlResult.data?.[queryType] as any[])[2][personTypeLowerFirst]).toBeUndefined();
+            expect((gqlResult.data?.[queryType] as any[])[0][SCORE_FIELD]).toBeGreaterThanOrEqual(
+                (gqlResult.data?.[queryType] as any[])[1][SCORE_FIELD]
+            );
+            expect((gqlResult.data?.[queryType] as any[])[1][SCORE_FIELD]).toBeGreaterThanOrEqual(
+                (gqlResult.data?.[queryType] as any[])[2][SCORE_FIELD]
+            );
+        });
+        test("Throws error if no return is requested", async () => {
+            const query = `
+                query {
+                    ${queryType}(phrase: "some name") {}
+                }
+            `;
+            const gqlResult = await graphql({
+                schema: generatedSchema,
+                source: query,
+                contextValue: {
+                    driver,
+                    driverConfig: { database: databaseName },
+                },
+            });
+
+            expect((gqlResult.errors as any[])[0].message).toStartWith("Syntax Error");
+        });
+        test("Throws error if no phrase argument", async () => {
+            const query = `
+                query {
+                    ${queryType} {
+                        ${personTypeLowerFirst} {
+                            name
+                        }
+                    } 
+                }
+            `;
+            const gqlResult = await graphql({
+                schema: generatedSchema,
+                source: query,
+                contextValue: {
+                    driver,
+                    driverConfig: { database: databaseName },
+                },
+            });
+
+            expect((gqlResult.errors as any[])[0].message).toBe(
+                `Field "${queryType}" argument "phrase" of type "String!" is required, but it was not provided.`
+            );
+        });
```

Comment:
I'm sorry to say it, but these tests could really all be removed - they're just testing the GraphQL query language which I don't really think we should be doing in our integration tests.

```suggestion

```